network/default.go

```diff
@@ -39,11 +50,18 @@ type network struct {
 	// client is network client
 	client client.Client
 
+	// ctrlClient is ControlChannel client
+	ctrlClient transport.Client
```

Comment:
Map of clients in case this grows? map[string]transport.Client with channel name as the key.

Reply:
Yeah. Good shout. Can't think of a use case as of now but `ControlChannel` might come in handy to hash over.

Reply:
Agree, can't think of a better name yet...